Shared/BookSortService+SortError+PlayListSortOrder.swift

```diff
@@ -31,6 +40,7 @@ enum SortError: Error {
 public enum PlayListSortOrder: String {
     case metadataTitle = "title"
     case fileName = "originalFileName"
+    case sortByMostRecent = "lastPlayDate"
```

Comment:
let's change `sortByMostRecent` to just `mostRecent`, so when calling the function `sort(by:type:...` we can have consistency with the other cases in this enum

Reply:
updated based on comments.